### PR TITLE
Preliminary review - conformance to RFC9110

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- preliminary review to comply with HTTP specifications RFC9110

## Notes

- payload / body -> content

### CODE REVIEW

File: versions/3.2.0.md [hunk 1]
```diff
@@ -891,7 +891,7 @@ Field Name | Type | Description
...
+<a name="operationRequestBody"></a>requestBody | [request content Object](#requestBodyObject) \| [Reference Object](#referenceObject) | The request content applicable for this operation.  The `requestBody` is fully supported in HTTP methods where the HTTP 1.1 specification [RFC7231](https://tools.ietf.org/html/rfc7231#section-4.3.1) has explicitly defined semantics for request contents.  In other cases where the HTTP spec is vague (such as [GET](https://tools.ietf.org/html/rfc7231#section-4.3.1), [HEAD](https://tools.ietf.org/html/rfc7231#section-4.3.2) and [DELETE](https://tools.ietf.org/html/rfc7231#section-4.3.5)), `requestBody` is permitted but does not have well-defined semantics and SHOULD be avoided if possible.
```
Comment (ioggstream) [inline versions/3.2.0.md:894]: reference https://datatracker.ietf.org/doc/html/rfc9110

File: versions/3.2.0.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ An OpenAPI definition can then be used by documentation generation tools to disp
 		- [Operation Object](#operationObject)
 		- [External Documentation Object](#externalDocumentationObject)
 		- [Parameter Object](#parameterObject)
-		- [Request Body Object](#requestBodyObject)
+		- [request content Object](#requestBodyObject)
```
Comment (ioggstream) [inline versions/3.2.0.md:44]: Here and elsewere, retain Request Body

File: versions/3.2.0.md [hunk 3]
```diff
@@ -1292,23 +1292,23 @@ content:
...
-<a name="requestBodyContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | **REQUIRED**. The content of the request body. The key is a media type or [media type range](https://tools.ietf.org/html/rfc7231#appendix-D) and the value describes it.  For requests that match multiple keys, only the most specific key is applicable. e.g. text/plain overrides text/*
-<a name="requestBodyRequired"></a>required | `boolean` | Determines if the request body is required in the request. Defaults to `false`.
+<a name="requestBodyDescription"></a>description | `string` | A brief description of the request content. This could contain examples of use.  [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
+<a name="requestBodyContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | **REQUIRED**. The content of the request content. The key is a media type or [media type range](https://tools.ietf.org/html/rfc7231#appendix-D) and the value describes it.  For requests that match multiple keys, only the most specific key is applicable. e.g. text/plain overrides text/*
```
Comment (ioggstream) [inline versions/3.2.0.md:1303]:
```suggestion
<a name="requestBodyContent"></a>content | Map[`string`, [Media Type Object](#mediaTypeObject)] | **REQUIRED**. The request content. The key is a media type or [media type range](https://tools.ietf.org/html/rfc7231#appendix-D) and the value describes it.  For requests that match multiple keys, only the most specific key is applicable. e.g. text/plain overrides text/*
```

**Comment by handrews** [comment]
@ioggstream thanks for this!  We are rearranging how our branches and directories are laid out which is about to make this PR invalid, so I'm closing it (and other 3.2.0 draft PRs).  However, issue #3800 is tracking this, and you're welcome to re-submit once the branching/directory changes described in #3677 are finalized and implemented.

Otherwise, issue #3800  is in the 3.2.0 milestone already so we'll definitely look at it, and it will have a link to this PR for us to consider when deciding how to handle it.  As you guessed with your later comments, we won't change "Request Body Object" to "Content Body Object" in a minor release, but we should consider the general vocabulary alignment.

**Pull request closed without merging**